notebooks/triposr-3d-reconstuction/triposr-3d-reconstruction.ipynb

```diff
@@ -0,0 +1,607 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #41.        scene_codes = model(image)
</pre><p>This line causes the following error:</p>

<pre class="ql-syntax" spellcheck="false">TypeError: TSR.forward() missing 1 required positional argument: 'device'
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1852/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>fixed</p>